### PR TITLE
Add styling to login register

### PR DESCRIPTION
**Pull request by OptimumCOW**
https://trello.com/c/XRwhVjYt

### CODE REVIEW

**Comment by OptimumCOW** [comment]
image
image
image
image

File: WhaleSpotting/Areas/Identity/Pages/Account/Register.cshtml [hunk 1]
```diff
@@ -13,30 +13,29 @@
...
-<h1>@ViewData["Title"]</h1>
+<h1 class="text-title">@ViewData["Title"]</h1>
 
 <div class="row">
-    <div class="col-md-4">
+    <div class="card-component">
         <form asp-route-returnUrl="@Model.ReturnUrl" method="post">
-            <h4>Create a new account.</h4>
-            <hr />
+            @*<h4 class="text-subtitle">Create a new account.</h4>*@
```
Comment (oskarwilliams) [inline WhaleSpotting/Areas/Identity/Pages/Account/Register.cshtml]: Commented code

File: WhaleSpotting/wwwroot/css/site.css [hunk 2]
```diff
@@ -1,65 +1,238 @@
...
+    font-size: 16px;
+}
+
+    .nav-link:hover {
+        background-color: #004785;
+        text-decoration: none;
+        color: white;
+        border-radius: 5px;
+        width: max-content;
+    }
```
Comment (oskarwilliams) [inline WhaleSpotting/wwwroot/css/site.css]: indented one more tab

File: WhaleSpotting/wwwroot/css/site.css [hunk 3]
```diff
@@ -1,65 +1,238 @@
...
+    transition: all 0.3s ease;
+    border: 1px solid #0072B5;
+    box-sizing: border-box;
+   
+}
+
+    .form-control:focus {
+        outline: unset;
+        border: 2px solid #004785;
+    }
```
Comment (oskarwilliams) [inline WhaleSpotting/wwwroot/css/site.css]: extra indent

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
Looks great!! Just a couple minor comments

File: WhaleSpotting/wwwroot/css/site.css (hunk 2 above)
Comment (OptimumCOW) [reply WhaleSpotting/wwwroot/css/site.css]: Addressed.

File: WhaleSpotting/wwwroot/css/site.css (hunk 3 above)
Comment (OptimumCOW) [reply WhaleSpotting/wwwroot/css/site.css]: Addressed.

File: WhaleSpotting/Areas/Identity/Pages/Account/Register.cshtml (hunk 1 above)
Comment (OptimumCOW) [reply WhaleSpotting/Areas/Identity/Pages/Account/Register.cshtml]: Addressed.

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**